src/App/components/DisplayColor/index.test.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { shallow } from "enzyme";
+
+import DisplayColor from "./index";
+
+describe("DisplayColor", () => {
```

Comment:
Should test that optional props work as intended (not included when not provided, included when provided)